tests/test.py

```diff
@@ -112,6 +113,9 @@
 
 TEST_WITH_TRACING = os.environ.get('TEST_TRACE', 'off').lower() == 'on'
 TEST_TRACE_TO_JAEGER = os.environ.get('TEST_TRACE_JAEGER', 'off').lower() == 'on'
+TEST_REDIS_KEYS_DB = 0
+TEST_REDIS_TOKEN_DB = 1
```

Comment:
can we move these variable to RedisMixin class level to not pollute global scope and allow to overwrite them in the inherited classes? because if some subclass will need change some of these values, it will change global variable for all test Classes, not only for itself. But if they assigned in the class level scope then it may be overrides in the subclass without touching global scope or parent class.

Reply:
Done:)